### DISCUSSION TITLE
I Want to connect My Domian from freenom to github, what it won't work, ibut it works for most peopl

### DISCUSSION THREAD

**Discussion by radioblahaj** [community/community#23344] [Other Feature Feedback, Questions, & Ideas]
<p>What Is Wrong?</p>
<p>What Is A Nameserver?</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p><a class="mention" href="/iamtregsthedev">@iamtregsthedev</a> Thanks for being a part of the GitHub Community Forum!</p>
<p>As a starting point, I would recommend checking out this GitHub Help article. It goes over some of the common troubleshooting tips that users see when trying to use a custom domain with a GitHub Pages site:</p>
<p><a href="https://help.github.com/articles/troubleshooting-custom-domains/" rel="nofollow noopener">https://help.github.com/articles/troubleshooting-custom-domains/</a></p>

**Comment by radioblahaj** [comment]
<blockquote>
<p><a class="mention" href="/nadiajoyce">@nadiajoyce</a> wrote:</p>
</blockquote>
<blockquote>
<p><a class="mention" href="/iamtregsthedev">@iamtregsthedev</a> Thanks for being a part of the GitHub Community Forum!</p>
</blockquote>
<blockquote>
<p>As a starting point, I would recommend checking out this GitHub Help article. It goes over some of the common troubleshooting tips that users see when trying to use a custom domain with a GitHub Pages site:</p>
</blockquote>
<blockquote>
<p><a href="https://help.github.com/articles/troubleshooting-custom-domains/" rel="nofollow noopener">https://help.github.com/articles/troubleshooting-custom-domains/</a></p>
</blockquote>
<p>Thanks!</p>

**Comment by Natemo2625** [comment]
<p>Even after reading through the troubleshooting tips, I still can’t get it to work, any ideas?</p>
